pkg/agent/testdata/RawUbuntu/CustomData

```diff
@@ -221,6 +221,8 @@ write_files:
     #!/bin/bash
 
     
+    iptables -t nat -A POSTROUTING -m iprange ! --dst-range 168.63.129.16 -m addrtype ! --dst-type local ! -d 10.0.0.0/8 -j MASQUERADE
```

Comment:
why do we update this?

Reply:
I updated this ContainerService.Properties.OrchestratorProfile.KubernetesConfig.NetworkPlugin to use Azure CNI specifically and we did not use ipmasqagent, so this rule is added per the definition of nodecustomdata.yaml. I guess it's the expected behavior.
```
{{if not IsIPMasqAgentEnabled}}
    {{if IsAzureCNI}}
    iptables -t nat -A POSTROUTING -m iprange ! --dst-range 168.63.129.16 -m addrtype ! --dst-type local ! -d {{GetParameter "vnetCidr"}} -j MASQUERADE
    {{end}}
{{end}}
```

Reply:
AKS default to have ipmasqagent, can you update the test data so it's closer to the actual scenairo?

Reply:
Sure

Reply:
Enable ip-masq-agent, we don't have such a rule anymore.